src/import-data/ImportDataOverview.ts

```diff
@@ -56,23 +53,13 @@ export const ImportDataOverview = (props: ImportDataOverviewProps): ReactNode =>
 
   return div({ style: styles.card }, [
     h2({ style: styles.title }, [getTitleForImportRequest(importRequest)]),
-    'url' in importRequest && div({ style: { fontSize: 16 } }, ['From: ', importRequest.url.hostname]),
+    h3({ style: { fontSize: 16 } }, ['Dataset source:']),
+    'url' in importRequest && div({ style: { marginTop: '1rem' } }, [`${importRequest.url.href.split('?')[0]}`]),
```

Comment:
If there's no source to show, we'll also want to hide the "Dataset source" header.

```suggestion
    'url' in importRequest && h(Fragment, [
      h3({ style: { fontSize: 16 } }, ['Dataset source:']),
      div({ style: { marginTop: '1rem' } }, [`${importRequest.url.href.split('?')[0]}`]),
    ])
```

Reply:
ha yup, was just looking into a way to do this without just copying/pasting the same condition :) thanks for the snippet!